tools/raspi_debug_server/oocd_raspberry.cfg

```diff
@@ -0,0 +1,11 @@
+telnet_port 4444
+gdb_port 3333
+bindto 0.0.0.0
+# raspberrypi
+interface sysfsgpio
+
+sysfsgpio_swd_nums 25 24
+#nRST = 18
```

Comment:
do we need the reset? or can we remove this line?

Reply:
We don't but It's wired

Reply:
It's not necessary and can be removed. it was a note in case nRST was needed.

Reply:
removed